src/obfl-specification.html

```diff
@@ -2618,6 +2618,21 @@ <h3 id="block-atts">Block</h3>
     <dd>Indent of every line in the block except the first one (in
     characters)</dd>
     <dd>A number.</dd>
+    <dd>This attribute is used to create hanging indentation.</dd>
+  <dt>right-last-line-indent [optional]</dt>
```

Comment:
Optional?

What does it mean that you add this tag without character number? Is this for when you have differing page number lengths so you need the system to figure out how much to indent?

Other than that this seems to be a really good change and I will accept it.

Reply:
I don't understand the question. These are all attributes for the block element, and all attributes are optional. If you don't specify `right-last-line-indent`, or any other indent or margin attribute, there is no indent/margin.